statApp/src/tblIFStat.cpp

```diff
@@ -125,12 +126,14 @@ struct IFStatTable : public StatTable {
             } else if(auto body = XRTA_CHECK(uint8_t, IFLA_LINKMODE, ratt)) {
                 tr.set("linkmode", *body);
 
+                // IFLA_CARRIER_UP/DOWN_COUNT were added in Linux 4.16
+#if LINUX_VERSION_CODE > KERNEL_VERSION(4,16,0)
```

Comment:
A minor point.  "were added in X" suggests to me a `>= X`.  "were added after X" would be `> X`.